From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: NaKyle <nakyle.wright@gmail.com>
Date: Thu, 5 Dec 2024 17:45:18 -0800
Subject: cec: cros-ec: update dmi match table to support newer bios releases

Signed-off-by: NaKyle <nakyle.wright@gmail.com>
(cherry picked from commit 4698ef7dff9621aec890b73e2ceebd663a0d1015)

OGC-tag: EXTERNALLY-MAINTAINED
---
 drivers/media/cec/platform/cros-ec/cros-ec-cec.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/media/cec/platform/cros-ec/cros-ec-cec.c b/drivers/media/cec/platform/cros-ec/cros-ec-cec.c
index be005a298c579ab30611ceaa3e6dd056ce07f1e4..06a20cdf64f7eec4dc654b2298df773fac45103e 100644
--- a/drivers/media/cec/platform/cros-ec/cros-ec-cec.c
+++ b/drivers/media/cec/platform/cros-ec/cros-ec-cec.c
@@ -310,8 +310,14 @@ static const char *const port_db_conns[] = { "Port D", "Port B", NULL };
 static const char *const port_ba_conns[] = { "Port B", "Port A", NULL };
 static const char *const port_ab_conns[] = { "Port A", "Port B", NULL };
 static const char *const port_d_conns[] = { "Port D", NULL };
+static const char *const port_c_conns[] = { "Port C", NULL };
 
 static const struct cec_dmi_match cec_dmi_match_table[] = {
+	/* AMD Lilac */
+	{ "AMD", "Lilac", "0000:03:00.0", port_c_conns },
+	/* Valve Fremont */
+	{ "OEM", "F7F", "0000:03:00.0", port_c_conns },
+	{ "Valve", "Fremont", "0000:03:00.0", port_c_conns },
 	/* Google Fizz */
 	{ "Google", "Fizz", "0000:00:02.0", port_b_conns },
 	/* Google Brask */
